src/Capture.cpp

```diff
@@ -72,25 +65,24 @@ void Capture::initBuffers() {
 void Capture::doJob(int playDbLevel) {
 	if( ! this->jackIsConnected())
 		throw QLE("JACK ports are not connected!");
-	if( ! this->jack->isIdle())
+	if( ! this->audioIo->isIdle())
 		throw QLE("JACK client isn’t IDLE now!");
 
 	this->maxResponse = 0.0;
 
-	JackInfo info;
+	AudioInfo info;
 	info.playBuf = this->playBuf;
 	info.capBuf = this->capBuf;
 	info.length = this->wavInfo->length;
 	info.playDb = playDbLevel;
 
-	info.rate = this->jack->getRate(); // suppress gcc warning
-
-	this->jack->start(info);
+	info.rate = this->audioIo->getRate(); // suppress gcc warning
 
-	while( ! this->jack->isIdle() )
-		usleep(100000); // 0.1 sec
+	this->audioIo->process(info);
 
-	this->maxResponse = this->jack->getMaxResponse();
+	for (unsigned i = 0; i < this->wavInfo->length; ++i) {
+		this->maxResponse = std::max(this->maxResponse, fabs(this->capBuf[i]));
```

Comment:
On my system, this doesn’t compile: error: no matching function for call to ‘max(float&, double)’

Reply:
Hmm, this error sounds strange. However, i will push an update trying to fix that.